### ISSUE TITLE
document citation does not include author name even if author is associated with a document

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/mep-django#455]
## testing notes
- citation block at the bottom of content pages should know include author names in lastname, first format
- person snippet in wagtail now requires last name and first name as separate fields; check that existing names were split properly, edit one, and create a new one
- edit pages to test that names display correctly with one author, two authors, three authors

## dev notes
- [x] convert person snippet to split out last name and first name
- [x] migration should split existing names on the _last_ space
- [x] modify citation to include author names if present; first author is last name first, all others are in normal order
- [x] make sure existing templates display name properly (possibly add a property for `name` that displays the content previously in one field)
- [x] check that author/editor are included in html <head> metadata

**Comment by thatbudakguy** [comment]
as per 12/4 mtg discussion this can wait until next milestone.

**Comment by clmahoney** [comment]
- YES, person snippets are now divided into "last name," "first name," chunks: I checked all the pre-existing ones: Kotin, Joshua; Budak, Nick; Koeser, Rebecca Sutton. I added my name and Ian's name. I edited Joshua to Josh, then back. 
- YES, the citation looks right. I added this citation with two people to the "Logbooks" page: 
![Screen Shot 2020-02-13 at 2 12 59 PM](https://user-images.githubusercontent.com/16673934/74469612-03c60e00-4e6b-11ea-9f02-2d323a4bc894.png)
- I added three people, it looked right. I added one, it looked good too!
